.github/ISSUE_TEMPLATE/os_validation.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Validate Operating System
+about: Request validation of an operating system
+title: 'Validate OS VERSION'
+labels: ["kind/os-validation"]
+assignees: ''
+
+---
+
+<!-- Thanks for helping us to improve K3S! We welcome all OS validation requests. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->
```

Comment:
```suggestion
<!-- Thanks for helping us to improve K3s! We welcome all OS validation requests. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->
```
For consistency's sake with the rest of the doc.